src/test/taglibtest.cpp

```diff
@@ -36,7 +33,8 @@ TEST_F(TagLibTest, WriteID3v2Tag) {
 
     // Write metadata -> only an ID3v2 tag should be added
     mixxx::TrackMetadata trackMetadata;
-    trackMetadata.refTrackInfo().setTitle("title");
+    trackMetadata.refTrackInfo().setTitle(
```

Comment:
The test directory doesn't matter for the title. See also below. Why did you touch this?